numbers/search.go

```diff
@@ -0,0 +1,58 @@
+package numbers
```

Comment:
I don't think this is needed now, but we are quickly reaching critical mass where it may made sense to split into `numbers` and `stats`